src/libcore/cmp.rs

```diff
@@ -95,6 +96,35 @@ use self::Ordering::*;
 /// assert!(b1 != b3);
 /// ```
 ///
+/// ##How can I implement it to compare with another type?
```

Comment:
Space between `##` and `How` and `it` => `PartialEq` imo

Reply:
That's not the issue. ;)